src/Field/Image.php

```diff
@@ -164,4 +174,28 @@ protected function fillMetadataFields(array $imageData)
         $this->height = $imageData['height'];
         $this->sizes = $imageData['sizes'];
     }
+
+    /**
+     * @param string|array      comma seperated string or an array
+     *
+     * @return string|array     string if only one meta key was received as input, otherwise an array with the values of all meta keys received as input
+     */
+    public function fetchCustomMetadataValues($metaKeys)
```

Comment:
Maybe this `metadata` method should be another method, and then refactor `fetchCustomMetadataValues()` one to accept both parameters.